src/bindings/python/tests/test_onnx/test_backend.py

```diff
@@ -101,16 +101,9 @@ def expect_fail(test_case_path, xfail):  # type: (str) -> None
     (
         xfail_issue_FLOAT_LIKE,
         "OnnxBackendNodeModelTest.test_cast_BFLOAT16_to_FLOAT_cpu",
-        "OnnxBackendNodeModelTest.test_cast_FLOAT16_to_DOUBLE_cpu",
```

Comment:
Great to see this issue resolved, Could you comment why the bfloat16 tests can't be enabled?

Reply:
It fails inside MKLDNNInferRequest::SetBlob bacause input precision and blob precisions don't match.